README.md

```diff
@@ -1 +1,248 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- DESCRIPTION VIDEO LINK -->
+## 🚀 Description Video Link <a name="live-demo"></a>
+
+- [Description Video Link](https://www.loom.com/share/98e3385a8dee47eeb2014bbbdc039ca3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Before you go ahead with this React tutorial, please make sure you have:
+
+- Basic understanding of HTML and CSS.
+- JavaScript fundamentals (object, array, conditionals etc).
+- Familiarity with JavaScript ES6 features (class syntax, arrow functions, object destructuring etc).
+
+2. [Node and npm](https://nodejs.org/en/download/) are installed. Here are the versions I'll be using while making this tutorial:
+
+    ```shell
+    $ node --version
+    v16.13.2
+
+    $ npm --version
+    8.1.2
+    ```
+    > Installing npm adds two commands to the system—`npm` and `npx`—both of which I'll be using while making this tutorial.
+
+3. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is installed. Here's the version I'll be using while making this tutorial:
+
+    ```shell
+    $ git --version
+    git version 2.29.1.windows.1
+    ```
+
+4. A [GitHub](https://github.com/signup) account. :octocat:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/developertariq/bookstore.git
+
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+npx create-react-app .
+npm install react-redux @reduxjs/toolkit
+npm install react-router-dom  
+npm i -S redux
+npm i -S prop-types  
+npm i axios 
+npm i react-test-renderer
+npm install react-bootstrap bootstrap
+npm i -S redux-thunk 
+npm i react-circular-progressbar
+npm i -S uuid
+npm install react-icons
+npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+npm install history
+npm install --save-dev jest
+npm install --save-dev babel-jest @babel/core @babel/preset-env
+npm install --save-dev jest @testing-library/react
+npm install --save-dev redux react-redux
+npm install --save-dev jest-environment-jsdom
+npm install --save-dev @testing-library/jest-dom
+npm i redux-mock-store
+```
+ -->
```

Comment:
- Let's remove this commented code to keep our readme clean.

Reply:
Hello @zieeco thank you for your comment. I have removed the commented code.